tools/autograd/gen_python_functions.py

```diff
@@ -915,7 +1128,56 @@ def is_smaller(d1, d2):
     return [decl for i, decl in sorted_deps]
 
 
-def get_python_signature(declaration, include_out):
+#
+# python signature codegen
+#
+
+SCHEMA_TYPE_CONVERSION_HACKS = {
+    'Type': 'ScalarType',
```

Comment:
I really doubt this ever does anything.

Reply:
It was firing, but only on instances of `Type` introduced elsewhere in the codegen :)

Reply:
that exists? :(